src/app/pages/registration/register-dead/register-dead.page.html

```diff
@@ -27,7 +27,7 @@
                     </ng-template>
                     <div *ngIf="images.length > 0, else elseblock" class="photos-container">
                         <div (click)="confirmPhotoRemoval(i)" *ngFor="let image of images; index as i" >
-                            <ion-img class="photo" *ngIf="capturedImage" [src]="'data:image/png;base64, ' +  images[i]"></ion-img>
+                            <ion-img class="photo" *ngIf="capturedImage" [src]="images[i]"></ion-img>
```

Comment:
Er det en grunn til at du velger å hente bilde med index ("images[i]") framfor å bare skrive "image"?

Reply:
jeg har lagt det til fra ts.filen i stedet for

Reply:
Altså hvorfor skriver du dette:
```
<ion-img class="photo" *ngIf="capturedImage" [src]="images[i]"></ion-img>
```
og ikke dette:
```
<ion-img class="photo" *ngIf="capturedImage" [src]="image"></ion-img>
```

Reply:
Det e ett fett da, du treng ikke å endre noe. Æ bare hadde lyst til å vær en dritt.